docs/modules/reference/pages/datasources/kafka.adoc

```diff
@@ -0,0 +1,92 @@
+
+= Kafka Datasource
+
+The Kafka datasource is used with a xref:admin:distributed_install.adoc[distributed installation of ALEC].
+It interfaces with OpenNMS by leveraging the event consumer and Kafka Producer features.
+
+The Kafka Producer forwards alarms, nodes, and network topology data that is encoded in OpenNMS protocol buffers to topics (see https://docs.opennms.com/horizon/31/operation/deep-dive/kafka-producer/kafka-producer.html[Kafka Producer] in the main documentation).
+The datasource consumes data from these topics and uses it to provide ALEC with a feed of alarms and inventory information.
+
+[topics]
+== Configure topics
+
+You can update the following properties to configure the names of individual Kafka topics:
+
+[options="header", cols="1,2,2"]
+|===
+| Property
+| Description
+| Default Value
+
+| alarmTopic
+| Alarms managed by OpenNMS.
+| alarms
+
+| alarmFeedbackTopic
+| Alarm feedback managed by OpenNMS.
+| alarmFeedback
+
+| edgesTopic
+| Topology managed by OpenNMS.
+| edges
+
+| nodeTopic
+| Node inventory managed by OpenNMS.
+| nodes
+
+| eventSinkTopic
+| Used to forward events to OpenNMS.
+| OpenNMS.Sink.Events
+
+| inventoryTopic
+| Used to maintain inventory state.
+| alec-inventory
+|===
+
+IMPORTANT: These topics must exist in the Kafka broker for the datasource to operate.
+
+== Event compatibility
+
+The `eventTimeFormat` property is used to determine how event creation times are encoded when ALEC sends events to OpenNMS.
+Its value should be configured depending on your OpenNMS version: for any version below Horizon 25.1.0 or Meridian 2019.1.0, the value should be `simple`.
```

Comment:
Verify the version numbers, as I think the latest ALEC requires much higher minimum version. Plus, I believe Meridian 2019 will be end of life when we go to Meridian 2023, but we also need to verify that.